revdep/README.md

```diff
@@ -1,29 +1,27 @@
 # Platform
 
-|field    |value                                                                             |
-|:--------|:---------------------------------------------------------------------------------|
-|version  |R version 4.2.0 (2022-04-22)                                                      |
-|os       |macOS Catalina 10.15.7                                                            |
-|system   |x86_64, darwin17.0                                                                |
-|ui       |RStudio                                                                           |
-|language |(EN)                                                                              |
-|collate  |en_US.UTF-8                                                                       |
-|ctype    |en_US.UTF-8                                                                       |
-|tz       |America/New_York                                                                  |
-|date     |2022-10-03                                                                        |
-|rstudio  |2022.07.1+554 Spotted Wakerobin (desktop)                                         |
-|pandoc   |2.18 @ /Applications/RStudio.app/Contents/MacOS/quarto/bin/tools/ (via rmarkdown) |
+|field    |value                                                                               |
+|:--------|:-----------------------------------------------------------------------------------|
+|version  |R version 4.2.0 (2022-04-22)                                                        |
+|os       |macOS Catalina 10.15.7                                                              |
+|system   |x86_64, darwin17.0                                                                  |
+|ui       |RStudio                                                                             |
+|language |(EN)                                                                                |
+|collate  |en_US.UTF-8                                                                         |
+|ctype    |en_US.UTF-8                                                                         |
+|tz       |America/New_York                                                                    |
+|date     |2022-12-06                                                                          |
+|rstudio  |2022.07.2+576 Spotted Wakerobin (desktop)                                           |
+|pandoc   |2.19.2 @ /Applications/RStudio.app/Contents/MacOS/quarto/bin/tools/ (via rmarkdown) |
 
 # Dependencies
 
 |package      |old   |new   |Δ  |
 |:------------|:-----|:-----|:--|
-|REDCapTidieR |0.1.2 |0.1.2 |   |
-|crayon       |NA    |1.5.2 |*  |
-|jsonlite     |NA    |1.8.2 |*  |
-|readr        |NA    |2.1.3 |*  |
-|vctrs        |NA    |0.4.2 |*  |
-|vroom        |NA    |1.6.0 |*  |
+|REDCapTidieR |0.1.3 |0.1.3 |   |
```

Comment:
This says "new: 0.1.3". I guess we need to rerun this after bumping the version?

Reply:
Yea probably, when we go for the actual submission today I'll bump and rerun and everything in `refresh.R`